test/integration/patients/worklist/reduced-schedule.js

```diff
@@ -100,6 +100,7 @@ context('reduced schedule page', function() {
       .wait('@routeActions')
       .itsUrl()
       .its('search')
+      .should('contain', 'include=patient')
       .should('contain', `filter[clinicians]=${ currentClinician.id }`)
       .should('contain', `filter[states]=${ stateTodo.id },${ stateInProgress.id }`)
       .should('contain', 'fields[flows]=name,state');
```

Comment:
Should we check `.should('contain', 'fields[patients]=first_name,last_name')` as well?

Reply:
we could.. I'm on the fence.. I'm not even convinced testing the include makes a ton of sense.  I think the `fields[flow]` check here comes from copy-paste of the worklist.. where it exists for actions but not flows.. but these things are just static.. no logic determines if these things show up..  which isn't necessarily a reason not to test it.. but might be overkill.. thoughts?

Reply:
I usually like to put things in the tests to stop us from deleting something in the future and having unintended consequences. But that doesn't apply to this. It's just static without any logic. 

So ya... I'm on the fence now too haha. Probably fine without it?